### ISSUE TITLE
Can't choose spell level when casting spells as Former Cultist (Scoundrel) or Templar (Warrior)

### ISSUE THREAD

**Issue by luiscastilho** [FLPub/symbaroum5ecore#66]
Hello,

Mystic class spellcasting works great. When you cast a spell you can choose at what level the spell is going to be cast and you get Corruption accordingly. But the same doesn't work for other spellcasting classes - Scoundrel's Former Cultist approach and Warrior's Templar approach.

Bellow is an example using a Templar character. Note that this character is a 3rd level Warrior so, following the Armored Mystic table (Templar feature), he/she should be able to cast 2nd level spells. Any time you cast a spell using one of these classes (Former Cultist or Templar) you get the error shown below ("Base spell level exceeds maximum spell level of this actor") and the "Cast at Level" drop-down only shows 1st Level.

Let me know if I'm missing something or doing something wrong. I don't have a lot of experience using Foundry.

Thank you

Templar's spell table (Armored Mystic feature):
![templar-spell-table](https://github.com/FLPub/symbaroum5ecore/assets/3590078/733a8e17-2b2c-43c9-b2e8-48aaa3f25b88)

Character features:
![templar-features](https://github.com/FLPub/symbaroum5ecore/assets/3590078/a7e3b954-9e8c-46d6-8b79-7b79fe28292a)

Casting a spell as this character:
![templar-cast-spell](https://github.com/FLPub/symbaroum5ecore/assets/3590078/88bae281-0f30-4bae-8943-73e7559931f6)

**Comment by trioderegion** [comment]
What version of the symbaroum module and dnd5e are you using?

**Comment by luiscastilho** [comment]
Forgot to include the most relevant information, of course.

DnD5e v2.3.1
RoS5e Core v3.0.2

No other systems installed but I do use a few other modules (below). I will disable them and test again just to make sure.

![image](https://github.com/FLPub/symbaroum5ecore/assets/3590078/6b25673b-71a7-4aa8-8487-7abd29adc48d)

**Comment by luiscastilho** [comment]
Same result. Only modules active were the RoS modules.
Let me know if I can help with anything else.

**Comment by trioderegion** [comment]
> Forgot to include the most relevant information, of course.
> 
> DnD5e v2.3.1 RoS5e Core v3.0.2
> 
> No other systems installed but I do use a few other modules (below). I will disable them and test again just to make sure.
> 

I think this is the root issue, actually. There is a beta branch for dnd5e 3.x. I would recommend making a backup and trying this again. I'm working on finalizing and getting a proper version out, for now, you can use this manifest for the beta: https://github.com/FLPub/symbaroum5ecore/releases/download/4.0%2B1/module.json

**Comment by luiscastilho** [comment]
OK, testing the version you sent now. I created a new world with DnD5e 3.1.2 and RoS Core v4.0+1. Other than that, all I have enabled are the RoS content modules (Player's Guide, GM's Guide and Bestiary).

I'm running into JavaScript errors while trying to create a PC. Sometimes it happens when trying to select a skill, sometimes when adding an armor proficiency or a tool proficiency.

```
TypeError: An error occurred while rendering Syb5eActorSheetCharacter 92. item.actor is undefined
```

I uploaded a video showing the error in case it helps: https://www.dropbox.com/scl/fi/p2qotukj69rxvd2urxv2u/RoS-javascript-error.mp4?rlkey=dk2vcpmkmqtneemnaswlz5p6e&st=gkigbfwb&dl=0

By the way this game is hosted on The Forge (forge-vtt.com). Not sure if this is relevant.

**Comment by luiscastilho** [comment]
Were you able to reproduce this problem? Let me know if I can help.

**Comment by trioderegion** [comment]
I have not had the chance to investigate, unfortunately. Its on the docket, just not enough hours in the day 😞 , I hope you understand.

**Comment by trioderegion** [comment]
Closed by #70
